roles/ocp4-cloud-ipi/tasks/install.yml

```diff
@@ -4,6 +4,8 @@
   shell:
     cmd: "{{ user_path }}/openshift-install create cluster --dir {{ user_path }} --log-level debug"
     creates: "{{ user_path }}/auth"
+  environment:
+    ARM_SKIP_PROVIDER_REGISTRATION: "true"
```

Comment:
I think we should make it conditional, especially if we want to push this upstream, as this is a special restriction, not the default setup

Reply:
fixed, please take a look